### PR TITLE
EIP2981 status changed to 'Last Call'. ✍️ 

### PR DESCRIPTION
**Pull request by blmalone**
fix: updating eip-2981 status from 'Review' to 'Last Call'.

### CODE REVIEW

**Review by seibelj — APPROVED** [review]

**Comment by blmalone** [comment]
Hey @MicahZoltu, do you know why this won't auto merge?

**Comment by MicahZoltu** [comment]
> - EIP 2981 state was changed from review to last call

You need an editor to review the change since the status is changing.  I don't handle ERCs, so you'll have to wait for another editor to check it out.

**Review by VexyCats — APPROVED** [review]

File: EIPS/eip-2981.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ eip: 2981
 title: NFT Royalty Standard
 author: Zach Burks (@vexycats), James Morgan (@jamesmorgan), Blaine Malone (@blmalone), James Seibel (@seibelj)
 discussions-to: https://github.com/ethereum/EIPs/issues/2907
-status: Review
+status: Last Call
+review-period-end: 2021-07-16
```
Comment (lightclient) [inline EIPS/eip-2981.md:7]:
```suggestion
review-period-end: 2021-07-22
```

**Review by lightclient — APPROVED** [review]
Looks good to me, thanks!

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 2981 state from Review to Last Call

**Comment by andrewschreiber** [comment]
Hey wanted to share the first NFT contract that implements ERC 2981 for posterity:
https://etherscan.io/address/0x4C77CD2764729D4F4596C420310A2eb89799aE07#readContract

**Pull request merged**